scripts/fetch-cl-contracts.py

```
@@ -0,0 +1,77 @@
+
+logger = logging.getLogger("scripts:fetch-cl-contracts")
+
+
+def main() -> int:
+    parser = argparse.ArgumentParser(description="Fetch CL FOP contract from IBKR.")
+    parser.add_argument("--env", choices=["dev", "prod"], default="dev")
+    args = parser.parse_args()
+
+    env_file = f".env.{args.env}"
+    if not os.path.exists(env_file):
+        raise FileNotFoundError(f"{env_file} not found")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)